.github/workflows/test.yml

```diff
@@ -112,3 +135,10 @@ jobs:
       - name: test
         run:  rake test TESTOPTS="-v --no-show-detail-immediately" OSSL_MDEBUG=1
         timeout-minutes: 5
+        if: ${{ !matrix.fips_enabled }}
```

Comment:
This condition is a temporary workaround. If you remove the `if`, you can reproduce the issue https://github.com/ruby/openssl/issues/603 on the fips mode case on CI.